### ISSUE TITLE
ESP8266iTachEmulator portion no longer builds (dependency issues)

### ISSUE THREAD

**Issue by jetenginesupreme** [probonopd/ESP8266iTachEmulator#5]
Hi @probonopd I just sent you a message on your build issue in the project's original repo.  I can't get this repo to build (dependency problems) via PlatformIO.  Could you please take a look?  Thank you.

**Comment by probonopd** [comment]
Hi @jetenginesupreme are you sure you can't build the code in _this_ repo? I have nothing to do with that other repo.

**Comment by jetenginesupreme** [comment]
Yes, certain.  I spent hours trying to sort dependency issues and missing header files (in this repo).  I am specifically trying to build the ESP8266iTachEmulator folder not the GUI. I would be glad to help you in any way to get this sorted.

**Comment by probonopd** [comment]
@theassassin do you have an idea why this might be the case? I think we had switched to PlatformIO exactly to pin all versions and avoid things like this?

@jetenginesupreme can you post _concrete_ error messages?
